tracker/tracker_debugger_api.yaml

```diff
@@ -1318,3 +1374,912 @@ components:
           description: Google Click ID
           type: string
           example: 1237abcdb
+    Log:
+      description: A tracking log
+      required:
+        - id
+        - time
+        - ip
+        - scheme
+        - method
+        - headers
+        - query_params
+      type: object
+      properties:
+        id:
+          description: UUID of the log
+          type: string
+          format: uuid
+          example: 76d1b36e-c2d5-44e8-ad60-028cb0bf17f6
+        time:
+          description: 'Time when the request was received by a tracker (in UTC)'
+          type: string
+          format: date-time
+          example: '2020-02-01T00:00:00.000000000Z'
+        ip:
+          description: 'Tracker client IP (real value can be provided in headers or `cip` parameter)'
+          type: string
+          format: ip
+          example: 75.13.13.87
+        host:
+          description: 'A host under which app is available'
+          type: string
+          example: piwikpro.tld
+        scheme:
+          description: 'Scheme used for the request'
+          type: string
+          example: http
+        method:
+          description: 'Method used for the request'
+          type: string
+          example: GET
```

Comment:
This could be an enum.

Reply:
Yes it could... but for now it's just a string...